Library/Formula/rem.rb

```diff
@@ -0,0 +1,15 @@
+class Rem < Formula
+  desc "Command-line tool to access OSX Reminders.app database"
+  homepage "https://github.com/kykim/rem"
+  url "https://github.com/kykim/rem/archive/20150618.tar.gz"
+  sha256 "e57173a26d2071692d72f3374e36444ad0b294c1284e3b28706ff3dbe38ce8af"
+
+  def install
+    xcodebuild
```

Comment:
Cool. No, no worries, sometimes `xcodebuild` by itself is fine but just wanted to check.

Can you add `depends_on :xcode => :build` one empty line below `sha256` and then we should be good to go on this.